superset/commands/report/execute.py

```diff
@@ -571,10 +586,13 @@ def _send(
                     recipient.type = ReportRecipientType.SLACKV2
                     notification = create_notification(recipient, notification_content)
                     notification.send()
-                except (UpdateFailedError, NotificationParamException) as err:
-                    # log the error but keep processing the report with SlackV1
-                    logger.warning(
-                        "Failed to update slack recipients to v2: %s", str(err)
+                except (UpdateFailedError, NotificationParamException) as mig_err:
```

Comment:
can you catch these errors below on line 597?

Reply:
This is actually inside an except block that's parallel to 597. But I can move that line to an "outer" try/except and have this specific except nested and I think that would remove this duplicated syntax. I'll try that now.